python/ctsm/modify_fsurdat/fsurdat_modifier.py

```diff
@@ -0,0 +1,100 @@
+"""
+Run this code by using the following wrapper script:
+tools/modify_fsurdat/fsurdat_modifier
+
+The wrapper script includes a full description and instructions.
+"""
+
+#  Import libraries
+import sys
+from getpass import getuser
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+
+from ctsm.modify_fsurdat.modify_fsurdat import ModifyFsurdat
+
+
+def get_parser():
+
+    """
+    Get parser object for this script.
+
+    Command-line inputs
+    - fsurdat_in: input file (str)
+    - fsurdat_out: output file (str)
+    - variable: variable to modify (str)
+    - value: value assigned to the variable to be modified (float)
```

Comment:
- [ ] Can delete this part of the comment, which is already out of date and will get more so

Reply:
Will eventually get to this (sorry for postponing).